gui/IrodsUpDownload.py

```diff
@@ -67,8 +67,7 @@ def _initialize_local_model(self):
             PyQt6.QtWidgets.QHeaderView.ResizeMode.ResizeToContents)
         home_location = PyQt6.QtCore.QStandardPaths.standardLocations(
             PyQt6.QtCore.QStandardPaths.StandardLocation.HomeLocation)[0]
-        index = self.localmodel.setRootPath(home_location)
-        self.localFsTreeView.setCurrentIndex(index)
```

Comment:
I think removing this causes the `LOCAL` FS tree view in this tab to default to the root entry, effectively `index = 0`.  The view in the ELabJournal tab is unaffected.

Reply:
That is correct. When we are done with the merging we look into how to open both the local and irods file tree correctly. this is just a quick patch.

Reply:
Removing `self.localFsTreeView.setCurrentIndex(index)` causes the root of the tree in the data transfer tab to be different from the root in the ELN tab.  Shouldn't they be consistent?

Reply:
Made the same changes in the elab tab and added todo's to discuss this in the future. If we indeed decide to make them similar we have some duplicate code to cleanup.

Reply:
I really thought the original view starting in the user's home directory was ideal and would like to extend that to the iRODS view.  The consistency I mentioned was between the local FS tree views in the different tabs, not the local FS and iRODS tree views.  Indeed, we can tackle that with Issue #4 or create a whole new issue for it.

Reply:
While I agree that having the tree open would be nice, we have to modify the way it does this to make it robust. Same problem as I encountered for the irods path where '<Zone>/home' is not accessible for all users. 
Unfortunately I could not quickly find the right commands for it.
At WUR we currently don't use the ELN tab, therefore I had not noticed that it has the same issue. 

So my proposition would indeed be to define what to do in this case when we pickup [#4](https://github.com/rijksuniversiteit-groningen/iBridges/issues/4)

Reply:
If the original solution for the local FS tree view would work than I would not have removed it... 
If it's something you need quickly than it could be part of the next branch. 

But as this is an Alpha version of iBridges I don't see the problem with removing this feature now and making a fix in a new branch.

Reply:
I'm sorry, but it _does_ work.  What does removing this one line fix?  I really don't understand.  That's why I'm continuing this conversation.

This is my view now:
![Screenshot 2023-02-22 at 10 24 42](https://user-images.githubusercontent.com/13783312/220578361-d314aa7f-c4ac-44f6-8bcb-0c6ea160aa68.png)

This was my view before the change:
![Screenshot 2023-02-22 at 10 29 14](https://user-images.githubusercontent.com/13783312/220579037-ea37fa4e-a870-4d41-9294-18d10a67c5a6.png)

I feel this original view far better until we can fix both views to our liking.

Reply:
Your right, it works for you just like the 'which' command worked... but that does not mean that it works in general.
That's the frustrating part of multiplatform development. 
It has to work on a lot of devices and configurations and just like '<zone>/home' is not always accesable 'C:\Users\username' is also not always accessible from the GUI.  

But this is clearly a blocker for you, and were getting no where like this...
so I will put it back, we finish the merge and than I will fix it in a new branch when also looking at the other trees.